lib/utils/testem-events.js

```diff
@@ -190,8 +190,8 @@ class TestemEvents {
         const moduleDetailFileName = path.join(this.root, `module-metadata-${currentDate}.json`);
         const sortedModuleMetadata = getSortedModuleMetaData(this.stateManager.getModuleMetadata());
 
-        writeJsonToFile(moduleDetailFileName, sortedModuleMetadata, { spaces: 2 });
-        ui.writeLine(`\nThis execution module details was recorded at ${moduleDetailFileName}`);
+        writeJsonToFile(moduleDetailFileName, Array.from(sortedModuleMetadata.values()),{ spaces: 2 });
+        ui.writeLine(`\nThese execution module details were recorded at ${moduleDetailFileName}`);
```

Comment:
I suggest remove `These` because it is not clear what `These` is referring to
`ui.writeLine(`\nExecution module details were recorded at ${moduleDetailFileName}`);`